disparse-core/src/main/java/disparse/parser/dispatch/CooldownMessage.java

```diff
@@ -0,0 +1,7 @@
+package disparse.parser.dispatch;
+
+public enum CooldownMessage {
```

Comment:
I wonder if CooldownMessage name is good, it's a bit confusing when we use things like `String cooldownMessage` for the actual message we want to send, versus this enum.  Maybe something like `CooldownStrategy` would be more correct in this case.  Thoughts?

Reply:
Agreed, _strategy_ is a much more helpful wording to that